### PR TITLE
Feature/video detail connect server

### PR DESCRIPTION
**Pull request by Darmody**
Based on #30

### CODE REVIEW

File: src/components/VideoItem/index.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import CSSModules from 'react-css-modules'
 import Icon from 'components/Icon'
 import styles from './styles.scss'
 
+const bref = (text, size = 150) => (
+  text.length > size ? `${text.substr(0, size)}...` : text
+)
```
Comment (calebx) [inline src/components/VideoItem/index.js]: overdo i think. 
`"hello".substr(0, 100) == "hello"`

File: src/constants/ActionTypes.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const APP_PREFIX = '@@JIGOJIGO'
+
+export const VIDEO_FETCH_REQUEST = `${APP_PREFIX}/VIDEO/FETCH/REQUEST`
```
Comment (calebx) [inline src/constants/ActionTypes.js]: `VIDEOS` ?

Comment (Darmody) [inline src/constants/ActionTypes.js]: prefer 单数

Comment (calebx) [inline src/constants/ActionTypes.js]: OK as well

File: src/components/VideoItem/index.js [hunk 3]
```diff
@@ -1,30 +1,35 @@
...
+
 const VideoItem = ({ video }) => (
   <div className="callout" styleName="video-item">
     <div className="media-object">
       <div className="media-object-section" styleName="cover-container">
-        <a href="#link">
+        <Link to={`/videos/${video.youtubeId}`}>
```
Comment (calebx) [inline src/components/VideoItem/index.js]: `title={video.title}` as below Link?

Comment (Darmody) [inline src/components/VideoItem/index.js]: OK

**Pull request closed without merging**